extra_settings/migrations/0009_alter_value_json.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 5.1.6 on 2025-02-07 14:13
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("extra_settings", "0008_auto_20221024_0717"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="setting",
+            name="value_json",
+            field=models.JSONField(blank=True, default=dict, verbose_name="Value"),
+        ),
+    ]
```

Comment:
I don't think we need a new migration, modifying the old one should be enough considering that the new field is backward compatible at database level.